properties/math.py

```diff
@@ -126,12 +141,13 @@ def validate(self, instance, value):
         """Determine if array is valid based on shape and dtype"""
         if not isinstance(value, (tuple, list, np.ndarray)):
             self.error(instance, value)
-        value = self.wrapper(value)
-        if not isinstance(value, np.ndarray):
-            raise NotImplementedError(
-                'Array validation is only implmented for wrappers that are '
-                'subclasses of numpy.ndarray'
-            )
+        if self.coerce:
+            value = self.wrapper(value)
+        valid_class = np.ndarray
+        if isinstance(self.wrapper, type):
+            valid_class = self.wrapper
```

Comment:
Maybe `valid_class = self.wrapper if isinstance(self.wrapper, type) else np.ndarray`?

Reply:
Hah, I had that exact thing, but it made the line too long... I'll play around with it a little more.